DisguiseUnityRenderStream/Runtime/DisguiseTextures.cs

```diff
@@ -51,38 +51,11 @@ public static Texture2D CreateTexture(int width, int height, RSPixelFormat forma
 
             if (texture != null)
             {
+                texture.hideFlags = HideFlags.HideAndDontSave;
                 texture.name = name;
             }
             
             return texture;
         }
-        
-        public static Texture2D ResizeTexture(Texture2D texture, int newWidth, int newHeight, RSPixelFormat format)
```

Comment:
Removed the resize codepath because it's not being used (DisguiseRenderStream.CreateStreams re-creates the texture sending objects).